source/rtos_none/rtos_api.h

```diff
@@ -0,0 +1,119 @@
+/**
+ * @file    rtos_api.h
+ * @brief   
+ *
+ * DAPLink Interface Firmware
+ * Copyright (c) 2019, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __RTOS_API_H__
+#define __RTOS_API_H__
```

Comment:
Should use the CMSIS-RTOS2 API header, but only implement those APIs which you need. This way you're sharing a single common API between no-rtos and rtos that won't get out of sync accidentally.